README.md

```diff
@@ -1,7 +1,39 @@
 # Internet Freedom Hack
-
 Currently in development at https://internetfreedomhack.netlify.com
 
 Continuously deployed via Netlify.
 
 Based on [Type Theme](https://rohanchandra.github.io/type-theme/)
+
+---
+## Development setup
+1. Install Ruby environment manager
+```shell
+brew install rbenv
```

Comment:
Worth noting this is MacOS only?

Other than that, looks good to me, although I should note that in my case I didn't do any ruby version management, the gems we're using are pretty flexible (so I got away with just doing the last two lines).

Reply:
Yup I will add MacOS in brackets after the title. Regarding the ruby version system, you are right that we don't need all of those previous steps but I think its good practice to use `rbenv` to manage different ruby versions for future projects in case we need it. What do you think?

Reply:
Definitely no harm having those lines in there, people like me will skip them if they want to.